src/components/styledSelect/styledSelect.module.css

```diff
@@ -0,0 +1,73 @@
+.root {
+  font-family: 'Quattrocento Sans', Arial, Helvetica, sans-serif;
+  background-color: #fff;
+  font-size: 1rem;
+  margin: auto 0;
+  width: 228px;
+  position: relative;
+}
+
+.header {
+  border: 1px solid #5f5f5f;
+  border-radius: 3px;
+  display: flex;
+  justify-content: space-between;
+  grid-template-columns: 8fr 1fr;
+}
+
+.headerText {
+  font-family: 'Quattrocento Sans', Arial, Helvetica, sans-serif;
+  font-size: 1rem;
+  color: #5f5f5f;
+  padding: 0 8px;
+  margin-top: 8px;
+  margin-bottom: 8px;
+  height: 1rem;
+  line-height: 1;
+}
+
+.trigger {
+  background-color: var(--button-background-color);
+  color: var(--button-text-color);
+  margin: 0;
+  border-left: 1px solid var(--button-border-color);
+  border-right: none;
+  border-top: none;
+  border-bottom: none;
+  cursor: pointer;
+  border-radius: unset;
```

Comment:
I had to change this to make it look the same in my dev environment as it did in Storybook - otherwise the corners of the button were too rounded compared to the parent and it looked wonky. I'm still not sure why it looked different given it was the same browser and same machine and everything.